Tasks/google-play-release/GooglePlay.ts

```diff
@@ -108,13 +108,29 @@ async function run() {
         let requireTrackUpdate = false;
         if (shouldUploadApks) {
             tl.debug(`Uploading ${apkFileList.length} APK(s).`);
-
             requireTrackUpdate = true;
+
+            let tryUploadMappingFile = tl.getBoolInput('shouldUploadMappingFile', false);
             for (const apkFile of apkFileList) {
                 tl.debug(`Uploading APK ${apkFile}`);
                 const apk: googleutil.Apk = await googleutil.addApk(edits, packageName, apkFile);
                 tl.debug(`Uploaded ${apkFile} with the version code ${apk.versionCode}`);
                 apkVersionCodes.push(apk.versionCode);
+
+                if (tryUploadMappingFile) {
+                    const mappingFilePattern = tl.getPathInput('mappingFilePath', false);
+                    tl.debug(`Mapping file pattern: ${mappingFilePattern}`);
+
+                    const mappingFilePath = resolveGlobPath(mappingFilePattern);
+                    tl.checkPath(mappingFilePath, 'mappingFilePath');
```

Comment:
I'm not familiar with this API. What happens if the glob pattern finds multiple mapping.txt files? e.g. `**/mapping.txt` finds `apk1/mapping.txt` and `apk2/mapping.txt`. Does it always return the first one?

My original implementation made an assumption that the path directly specifies the mapping.txt file related to the main APK, and I believe that's what will happen here too but wanted to check.

Reply:
It returns the one.  It uses the `glob.sync()` method and returns the first result.  You can also specify the full path to the mapping as in your original implementation.  I figure that 'most' projects will just have the one APK file, and likely also just have the one mapping file, so allowing glob results would help in set up for the common case.

Reply:
Sounds good to me!